style.css

```diff
@@ -0,0 +1,35 @@
+body {
+    margin: 0;
+  }
+
+.main-section{
+   font-family: Arial, Helvetica, sans-serif;
+}
+  
+  ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    width: 25%;
+    background-color: #f1f1f1;
+    position: fixed;
+    height: 100%;
+    overflow: auto;
+  }
+  
+  li a {
+    display: block;
+    color: #000;
+    padding: 8px 16px;
+    text-decoration: none;
+  }
+  
+  li a.active {
+    background-color: #04AA6D;
```

Comment:
Please change your hexadecimal color code from #04AA6D to #04aa6d to avoid linter errors.